src/index.html

```diff
@@ -1,13 +1,73 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__mac-image"
+          data-qa="hover"
+        >
+      </div>
+
+      <div class="card__title">
+        <h6 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h6>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+      </div>
+
+      <div class="card__stats">
+        <div class="stars-N">
+          <div class="stars-N__star"></div>
+          <div class="stars-N__star"></div>
+          <div class="stars-N__star"></div>
+          <div class="stars-N__star"></div>
+          <div class="stars-N__star"></div>
+        </div>
+        <div class="card__reviews">
+          <p class="card__review">
+            Reviews: 5
+          </p>
+        </div>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-name">
+          Price:
+        </p>
+
+        <p class="card__price-cost">
+          <strong>$2,199</strong>
+        </p>
+      </div>
+
+      <a
+        href="#"
+        data-qa="hover"
```

Comment:
you have a couple of data-qa="hover" attributes on the page. It may be the case why one test is failed